conf/core/ConfigOptions.asciidoc

```diff
@@ -3358,6 +3358,13 @@ If the script runs longer than the specified time, then it is forcefully stopped
 the value is set to -1, then there is no time limit for script tests.  This is useful when
 debugging tests which may hang on a remote build server.
 
+=== search.radius.calculator.element.criterion
```

Comment:
This setting allows the SearchRadiusCalculator to filter down by feaure type.  Expected to be set in the Generic Conflation script.  There is some redundancy with this and isMatchCandidate in the script for now, unfortunately.  See #3047.